src/adapter/src/coord/ddl.rs

```diff
@@ -217,14 +217,24 @@ impl<S: Append + 'static> Coordinator<S> {
                 .unwrap_or(SYSTEM_CONN_ID),
         )?;
 
+        // This will produce timestamps that are guaranteed to increase on each
+        // call, and also never be behind the system clock. If the system clock
+        // hasn't advanced (or has gone backward), it will increment by 1. For
+        // the audit log, we need to balance "close (within 10s or so) to the
+        // system clock" and "always goes up". We've chosen here to prioritize
+        // always going up, and believe we will always be close to the system
+        // clock because it is well configured (chrony) and so may only rarely
+        // regress or pause for 10s.
+        let oracle_write_ts = self.get_local_write_ts().await.timestamp;
```

Comment:
We may also want to include a short comment about how `self.send_builtin_table_updates` below on line 248 will use a timestamp larger than this one and in the future we hope to be able to use the same one.